frontend/src/container/GridCardLayout/GridCard/FullView/index.tsx

```diff
@@ -96,7 +97,7 @@ function FullView({
 		},
 		{
 			queryKey: `FullViewGetMetricsQueryRange-${selectedTime.enum}-${globalSelectedTime}-${widget.id}`,
-			enabled: !isDependedDataLoaded,
+			enabled: !isDependedDataLoaded && widget.panelTypes !== PANEL_TYPES.LIST,
```

Comment:
Please add comments around the logic here.

Reply:
Done ✅